sklego/model_selection.py

```diff
@@ -49,14 +61,17 @@ def __init__(
         self.train_duration = train_duration
         self.valid_duration = valid_duration
         self.gap_duration = gap_duration
+        self.n_splits = n_splits
+        self.maximise_training_set = False
```

Comment:
Any reason to not add this as an argument to `__init__`? Also, I think changing it into something like `window_mode='rolling'` and `window_mode='expanding'` is a bit more clear. @koaning?

Reply:
In the Issue someone suggested to not add another parameter to the API to not crowd it, but I like the window_mode Idea

Reply:
If the object has 7 inputs I might agree, but here the alternative is to add another cross validator that does have the feature which would be worse. I'm only wondering if `window_mode` is the best name. Why not just `window`?

Reply:
yep, `window` is probably better